packages/app/src/components/nominationInfo/EditCard.js

```diff
@@ -45,13 +46,15 @@ const EditCard = props => {
                           : new Date(dischargeDate)
                         : null
                     }
-                    render={({ field: { onChange, value } }) => (
+                    render={({ field: { onChange, value } }) => {
+                      return (
                       <DatePicker
                         selected={value}
                         onChange={onChange}
                         dateFormat='MM/dd/yyyy'
                       />
-                    )}
+                      )
```

Comment:
Nit: It took me staring at it for about 2 minutes because of the panoply of nested parens and braces, but we should probably indent this `DatePicker` now that it's wrapped in a return.

Reply:
fixed